operations/kubernetes/Manage_Sealed_Secrets.md

```diff
@@ -0,0 +1,181 @@
+
+
+## Manage Sealed Secrets
+
+Procedures for encrypting, decrypting, and modifying Kubernetes sealed secrets.
+
+Many of the examples in this section assume the USB stick used to install the system
+is still available. If site-init is no longer available on the USB stick and a backup
+has not been made, a new site-init will need to be created.
```

Comment:
This last sentence should make it clear where to look for how to create a new site-init.